### ISSUE TITLE
endpoint detection on the lookahead graph

### ISSUE THREAD

**Issue by daejeen** [alphacep/vosk-api#876]
Hi, All
Thanks for sharing a lots of experience for embedded speech recognition.
We made static and also dynamic graph model with own Korean sample DB.

My problem/questions are,

Static model works properly with following EndPoint Detection configures.
--endpoint.silence-phones=1:2:3:4:5:6:7:8:9:10
--endpoint.rule2.min-trailing-silence=0.5
--endpoint.rule3.min-trailing-silence=1.0
--endpoint.rule4.min-trailing-silence=2.0

**but dynamic graph does not work properly for rule4** 

Most of lookahead model has get same problem? or vosk api does not supports rule4?

Thanks in advance.

**Comment by nshmyrev** [comment]
Rule4 should still work. Could you provide an audio file demonstrating the problem?

**Comment by daejeen** [comment]
I've tried to solve my problem during last 4weeks.
I focused on AM because I think that it came from imperfect silence model.
But I realize that it came from LM not AM with my recent work.

My problem is that decoder of lookahead graph can not find endpoint in a short speech data as following.
[lookahead problem.txt](https://github.com/alphacep/vosk-api/files/8308404/lookahead.problem.txt)
Even if decoder could't find endpoint, It has to make a final text with rule4.

Anyway I make a new LM with a clean text, it works properly!

Thanks all,
